src/WopiValidator/Options.cs

```diff
@@ -20,6 +20,12 @@ class Options
 		[Option('l', "token_ttl", Required = true, HelpText = "WOPI access token ttl")]
 		public long AccessTokenTtl { get; set; }
 
+		[Option('r', "X-WOPI-UsingRestrictedScenario", Required = false, HelpText ="Restricted scenario")]
```

Comment:
Why does this need to be a command line option? This should be in the test definitions.

Reply:
For operation GetRestrictedLink and RevokeRestrictedLink, there's a required header "X-WOPI-UsingRestrictedScenario", but it's implementation specific so that I cannot put in test definitions. Similar for ReadSecureStore.

Reply:
I don’t follow, sorry. Are you saying that the entire test suite needs to be run with additional headers?

Reply:
Only test cases for GetRestrictedLink, RevokeRestrictedLink and ReadSecureStore, not the entire test suite. Value for header "X-WOPI-UsingRestrictedScenario" is implementation specific; Value for header "X-WOPI-ApplicationId" is environment related, it indicates ID of a target application stored in secure store. There should be a place to configure for the 2 headers when running test cases. Or can I add a config file in WopiValidator project used when running test cases for those 3 operations?